### DISCUSSION TITLE
Broken profile pages

### DISCUSSION THREAD

**Discussion by saltydk** [community/community#31244] [Other Feature Feedback, Questions, & Ideas]
Did someone push a change to profile pages that broke it? Looks collapsed on both Firefox and Edge.

**Comment by megasanjay** [comment]
It appears that only some people's profiles are broken. Mine is but most of my coworkers have normal pages 🤔

**Comment by airtower-luna** [comment]
Looks like the same issue described in #31242. :thinking:
